frontend/src/navigation/MainMenuBar.tsx

```diff
@@ -0,0 +1,76 @@
+import { AppBar, Box, Button, Menu, MenuItem, Toolbar, Typography } from '@mui/material';
+import { useState } from 'react';
+
+const options = ['Profile', 'Logout'];
+
+export default function MainMenuBar() {
+  const [anchorElUser, setAnchorElUser] = useState<null | HTMLElement>(null);
+
+  const handleOpenUserMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorElUser(event.currentTarget);
+  };
+
+  const handleCloseUserMenu = () => {
```

Comment:
Can we add a handle logout function, I have written one in NavBar.tsx

Reply:
Done!